.github/workflows/composer-lint.yml

```diff
@@ -0,0 +1,42 @@
+
+name: "Composer Lint"
+
+on:
+  workflow_call:
+    inputs:
+      php-version:
+        description: "The PHP version to use when running the job"
+        default: "8.0"
+        required: false
+        type: "string"
+    paths:
+      - "composer.json"
+
+jobs:
+  composer-lint:
+    name: "Composer Lint"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "${{ inputs.php-version }}"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v3"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: "none"
+          php-version: "${{ matrix.php-version }}"
+          tools: composer:v2, composer-normalize:2
+        env:
+          COMPOSER_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: "Composer normmalize"
+        run: "composer-normalize --dry-run"
+
+      - name: "Composer validate"
+        run: "composer validate"
```

Comment:
Doesn't composer-normalize already run this validation step?

Reply:
Even if that is not explicitly stated in [the docs](https://github.com/ergebnis/composer-normalize#details), I guess the validation is already covered (ergebnis/composer-normalize#297, ergebnis/composer-normalize#406).